sos/report/plugins/ultrapath.py

```diff
@@ -0,0 +1,40 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, RedHatPlugin
+
+
+class UltraPath(Plugin, RedHatPlugin):
```

Comment:
If / as also Ubuntu should run this plugin, I suggest replacing `RedHatPlugin` by `IndependentPlugin` (also in `import`, to let the plugin available on any distribution.

Reply:
ack, will make these changes.. thank you.